src/cpl/nuopc/rof_import_export.F90

```diff
@@ -489,184 +530,99 @@ end subroutine SetScalarField
   end subroutine fldlist_realize
 
   !===============================================================================
-
-  subroutine state_getimport(state, fldname, begr, endr, area, output, rc)
+  subroutine state_getimport(state, fldname, begr, endr, area, output, areacor, rc)
 
     ! ----------------------------------------------
     ! Map import state field to output array
     ! ----------------------------------------------
 
+    use ESMF, only : ESMF_StateGet, ESMF_FieldGet, ESMF_Field
+
     ! input/output variables
     type(ESMF_State)    , intent(in)    :: state
     character(len=*)    , intent(in)    :: fldname
     integer             , intent(in)    :: begr
     integer             , intent(in)    :: endr
     real(r8)            , intent(in)    :: area(begr:endr)
+    real(r8), optional  , intent(in)    :: areacor(:)
```

Comment:
As in https://github.com/ESCOMP/RTM/pull/24#discussion_r593500320 it seems problematic to have an optional argument appear before required arguments, here and in state_setexport. And as in https://github.com/ESCOMP/RTM/pull/24#discussion_r593500320, my preferred solution would be to have state_getimport directly use the module variable med2mod_areacor and state_setexport directly use mod2med_areacor.